bedrock/bedrock.py

```diff
@@ -216,10 +216,13 @@ def __init__(self, db, x, z, y):
       except KeyError:
         raise NotFoundError("Subchunk at {} {}/{} not found.".format(x, z, y))
       self.version, data = data[0], data[1:]
-      if self.version != 8:
+      if self.version not in [8, 9]:
         raise NotImplementedError("Unsupported subchunk version {} at {} {}/{}".format(self.version, x, z, y))
       numStorages, data = data[0], data[1:]
 
+      if self.version == 9:
+        self.y_db, data = data[0], data[1:]

```

Comment:
You should set `self.y_db` to `None` if the version is not 9, better to have a consistent set of properties even if they may be None than make the client deal with catching exceptions.